README.md

```diff
@@ -1,5 +1,9 @@
+[![Build Status](https://travis-ci.org/PenneyGadget/lookingfor.svg?branch=master)](https://travis-ci.org/PenneyGadget/lookingfor)
```

Comment:
@rrgayhart Ah, this is a problem and probably means it's not working. I no longer have access to the repo via https://travis-ci.org/ - as the admin I think you have to set that up now. If you create a Travis account and sync it with github you should be able to see all your repos and "turn on" LookingFor. Would you mind looking into that and reporting back?